extension/src/processExecution.ts

```diff
@@ -59,16 +58,3 @@ export const executeProcess = async (
   const { stdout } = await createProcess(options)
   return stdout
 }
-
-const sendOutput = (process: Process) =>
-  process.all?.on('data', chunk =>
-    Logger.log(chunk.toString().replace(/(\r?\n)/g, ''))
-  )
-
-export const createProcessWithOutput = (options: ProcessOptions) => {
```

Comment:
[F] moving this removed the need for a spy which was breaking some tests. It is currently only used in a single place so was ok to move.